ragna/assistants/_api.py

```diff
@@ -11,12 +11,10 @@
 from ragna.core import Assistant, EnvVarRequirement, RagnaException, Requirement, Source
 
 
-class ApiAssistant(Assistant):
-    _API_KEY_ENV_VAR: str
-
+class _ApiAssistant(Assistant):
     @classmethod
     def requirements(cls) -> list[Requirement]:
-        return [EnvVarRequirement(cls._API_KEY_ENV_VAR), *cls._extra_requirements()]
+        return []
```

Comment:
Should we make the accessibility of the API an availability check? Does the API have an endpoint that we could call to indicate whether the API is ready?

I would put that into 

```
@classmethod
def is_available():
```

similar to what we did [here](https://github.com/Quansight/ragna/blob/1d4cd568bdbd1335da39743743d5f2f48d05ca26/ragna/assistants/_airoboros.py#L19-L27).